meta/store.go

```diff
@@ -519,7 +535,9 @@ func (s *Store) createLocalNode() error {
 	}
 
 	// Set ID locally.
+	s.mu.Lock()
```

Comment:
This was detected as a race by our race detector.  Likely only due to test code and would not be a race condition in production, but better safe than sorry.